src/elements/content-uploader/ItemAction.js

```diff
@@ -101,7 +101,12 @@ const ItemAction = ({
         <div className="bcu-item-action">
             {tooltip ? (
                 <Tooltip position="top-left" text={intl.formatMessage(tooltip)}>
-                    <PlainButton onClick={onClick} type="button" isDisabled={status === STATUS_STAGED}>
+                    <PlainButton
+                        aria-label={intl.formatMessage(tooltip)}
```

Comment:
not necessary to change but could destruct `intl` near the top of the function:

`const { formatMessage } = intl;`

Reply:
did it!